### PR TITLE
create component ButtonHeart

### PR DESCRIPTION
**Pull request by DmitriyUshkvarok**

### CODE REVIEW

File: src/components/Heart/ButtonHeart.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const ButtonHeart: React.FC<ButtonHeartProps> = ({
+  active,
+  onClick,
+  className,
+}) => {
+  const imageUrl = active ? '/heart_full.svg' : '/heart_empty.svg'
+
+  return (
+    <div
```
Comment (Wdyffs) [inline src/components/Heart/ButtonHeart.tsx:12]: Use button here and set onClick on it

File: src/components/Heart/ButtonHeart.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      className={`duration-400 flex h-[54px] w-[54px] cursor-pointer items-center justify-center rounded-full bg-secondary transition ease-in-out hover:bg-[#D9D9D9] ${className}`}
+    >
+      <button onClick={onClick} type="button">
```
Comment (Wdyffs) [inline src/components/Heart/ButtonHeart.tsx:15]: Remove this button tag

**Review by Wdyffs — CHANGES_REQUESTED** [review]

File: src/components/Heart/ButtonHeart.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image'
+import { ButtonHeartProps } from './index'
+
```
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:3]: fix

File: src/components/Heart/ButtonHeart.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+const ButtonHeart: React.FC<ButtonHeartProps> = ({
```
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:4]: fix

File: src/components/Heart/index.ts
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/index.ts:1]: fix

File: src/components/Heart/ButtonHeart.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  active,
+  onClick,
+  className,
+}) => {
+  const imageUrl = active ? '/heart_full.svg' : '/heart_empty.svg'
```
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:9]: fix

File: src/components/Heart/ButtonHeart.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+  return (
+    <div
+      className={`duration-400 flex h-[54px] w-[54px] cursor-pointer items-center justify-center rounded-full bg-secondary transition ease-in-out hover:bg-[#D9D9D9] ${className}`}
```
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:13]: fix

File: src/components/Heart/ButtonHeart.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    >
+      <button onClick={onClick} type="button">
+        <Image
+          src={imageUrl}
+          alt={`heart ${active ? 'full' : 'empty'}`}
+          width={27}
+          height={27}
```
[the remark this answers is not in the document]
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:20]: fix

File: src/components/Heart/ButtonHeart.tsx (hunk 2 above)
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:15]: fix

File: src/components/Heart/ButtonHeart.tsx (hunk 1 above)
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:15]: fix

File: src/components/Heart/ButtonHeart.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  className?: string
+}
+
+const ButtonHeart = ({ active, onClick, className }: ButtonHeartProps) => {
+  const imageUrl = active ? heartFull : heartEmpty
+
+  return (
+    <button
+      onClick={onClick}
+      className={`duration-400 hover:bg-hover-heart flex h-[54px] w-[54px] cursor-pointer items-center justify-center rounded-full bg-secondary transition ease-in-out ${className}`}
```
Comment (Wdyffs) [inline src/components/Heart/ButtonHeart.tsx:17]: use className={twMerge('blablasba', className)}

**Review by Wdyffs — CHANGES_REQUESTED** [review]

File: src/components/Heart/ButtonHeart.tsx (hunk 8 above)
Comment (DmitriyUshkvarok) [reply src/components/Heart/ButtonHeart.tsx:17]: fix

**Review by Wdyffs — DISMISSED** [review]

**Review by Wdyffs — APPROVED** [review]

**Pull request merged**